src/encoder.rs

```diff
@@ -347,6 +347,24 @@ fn write_run<W: Write>(
     Ok(())
 }
 
+/// Allows fine-tuning some encoder parameters.
+///
+/// Pass to [`WebPEncoder::set_params()`].
+#[non_exhaustive]
```

Comment:
I wasn't sure if I should mark this #[non_exhaustive].

Reply:
Yes, it's a good idea to have non-exhaustive in public APIs.

Reply:
I agree, but it makes using this struct a bit less ergonomic, because
```
EncoderParams {
    use_predictor_transform: false,
    ..Default::default()
}
```
(like in the test in my second commit) is forbidden outside of the `image-webp` crate with non-exhaustive structs. It needs to be
```
let mut p = EncoderParams::default();
p.use_predictor_transform = false;
```
instead.

Reply:
You can make `EncoderParams::with_predictor_transform(false)`